meson.build

```diff
@@ -0,0 +1,27 @@
+project('crypto', 'c', version: '0.0.1', license: 'MIT')
+cc = meson.get_compiler('c')
+
+
+
+gmpDep = dependency('gmp', required: get_option('USE_GMP'))
+
+if gmpDep.found() and get_option('USE_GMP')
+  add_global_arguments('-DUSE_GMP=true', language: 'c')
+endif
+
+mDep = cc.find_library('m', required: true)
```

Comment:
Should be `required: false`, see https://mesonbuild.com/howtox.html#add-math-library-lm-portably